src/CabanaPD_Input.hpp

```diff
@@ -72,7 +72,7 @@ class Inputs
     }
 
     // Get a single input.
-    auto operator[]( std::string label ) { return inputs[label]; }
+    auto operator[]( std::string label ) { return inputs[label]["value"]; }
```

Comment:
@pabloseleson this is the issue - the overloaded [] directly returns the value once you made this change. I may keep it, but this obviously breaks when we try to index `["value"]` from a double instead of the json object

Reply:
What is the solution to this then?

Reply:
See 2de14c17598c3cd2a3e3150745e5f2fa9be6e965